docs/CHEAT_SHEET.md

```diff
@@ -18,7 +18,7 @@ HTML attributes with the following syntax have a special meaning to Qwik.
 | `key:=JSON`         | State of the `Entity` identified by `key`, serialized into JSON.                                                        |
 | `:.=JSON`           | Component state, serialized into JSON.                                                                                  |
 
-### JSX Components
+### JSX Components
```

Comment:
What is changing here?

Reply:
TBH, I'm not sure what happening here. 😓 

But this change fixes [current](https://github.com/BuilderIO/qwik/blob/main/docs/CHEAT_SHEET.md) rendering output.

![image](https://user-images.githubusercontent.com/6259812/148472842-e84e15da-f3b1-4c4b-97a0-74ceeb6badb1.png)

Reply:
The original character following the `###` has its high bit set (`U+00a0`) when what is required is a simple ASCII space.